docs/en/aws-firehose/aws-firehose-setup.asciidoc

```diff
@@ -43,15 +43,28 @@ In {kib}, navigate to *Management > Integrations* in the sidebar.
 Find the *AWS* integration by searching or browsing the catalog.
 
 [role="screenshot"]
-image::images/firehose-integrations-page.png[Integrations catalogue with the "AWS" integration highlighted]
+image::images/aws-integrations-page.png[Integrations catalogue with the "AWS" integration highlighted]
  
 Navigate to the *Settings* tab and click *Install AWS assets*.
 Confirm by clicking *Install AWS* in the popup. 
  
 [role="screenshot"]
 image::images/firehose-integrations-install-assets.png[AWS integration settings page with the "Install AWS assets" button highlighted]
+
+We also added an integration called Amazon Kinesis Data Firehose to automatically route AWS service logs to the respective integrations, which include field mappings, ingest pipelines, predefined dashboards and ect.
```

Comment:
i think we probably need to revisit this guide in general when we add instructions for the new routing integration. let's leave it out for now, if that's ok?

we can start to draft some better onboarding instructions with the new package in a seperate change.

Reply:
Will do! Sorry for some reason I missed your comments for 2 weeks 😂